pyvista/plotting/colors.py

```diff
@@ -451,6 +451,14 @@ class Color:
 
     """
 
+    # Supported names for each color channel.
+    CHANNEL_NAMES = [
+        {'red', 'r'},  # 0
+        {'green', 'g'},  # 1
+        {'blue', 'b'},  # 2
+        {'alpha', 'a', 'opacity'},  # 3
+    ]
```

Comment:
Slight nitpick: this should probably be a tuple, and we might even consider making the entries frozensets. Not that users would intentionally mutate it, but perhaps it's worth guarding against it. Not sure.

Reply:
I agree with the tuple, the frozensets I'm not really convinced. If a user really wants to tinker with that, just let them do so as it's not really 'harmful' in any way I believe.

Reply:
Yeah, good point.